service/templates/admin.html

```diff
@@ -2504,6 +2505,62 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 								</md-card-content>
 							</md-card>
 						</section>
+						<section layout="row" flex>
```

Comment:
Would this section show up in main staging or prod? Looking at the code, it looks like if a overrides section is attempted to be saved it uses the local database. Would this cause a panic or anything for those environments? 

To prevent it from showing, we could hide it behind a bool and the bool is if the shared db exists or not (if it's an issue)

Reply:
This is such a good idea!
done